en-US/firefox/privacy-hub.lang

```diff
@@ -0,0 +1,223 @@
+## NOTE: Demo page at https://www-demo1.allizom.org/firefox/privacy/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/privacy/
+
+
+;Privacy
+Privacy
+
+
+;Our Promise
+Our Promise
+
+
+;Our Products
+Our Products
+
+
+;Read the <a href="%s">Privacy Notice</a> for our products
+Read the <a href="%s">Privacy Notice</a> for our products
+
+
+# These strings are used in /firefox/privacy/
+# HTML page title
+;Firefox Privacy Promise
+Firefox Privacy Promise
+
+
+# HTML page description
+;Firefox takes less data, keeps it safe, and with no secrets.
```

Comment:
Seems like missing a verb here `... with no secrets.` (after takes... keeps...

Reply:
This is the string provided by the copywriter - let me check back on this.

Reply:
@peiying2 I’ll add a comment here with an alternate string for localisers